baseapp/baseapp/activity_log/context.py

```
@@ -0,0 +1,13 @@
+import pghistory
+
+from .models import VisibilityTypes
+
+
+def set_public_activity(verb: str, **kwargs: Any) -> None:
```

Ruff

Undefined name `Any`

(F821)